Libraries/Components/DatePicker/DatePickerIOS.ios.js

```diff
@@ -118,18 +128,24 @@ var DatePickerIOS = React.createClass({
 
   render: function() {
     var props = this.props;
+    let minimumDate = props.minimumDate;
```

Comment:
I just check the code, and I don't think we can set `minimumDate` and `maximumDate` in _onChange method, it's static values, so I think they needs to stay in render